infer-web.py

```diff
@@ -18,6 +18,14 @@
 import gradio as gr
 import numpy as np
 import torch
+try:
+    import intel_extension_for_pytorch as ipex # pylint: disable=import-error, unused-import
```

Comment:
Maybe you can move the ipex check to config.py and pass the test result to a variable in Config class ex. `has_xpu`

Reply:
I can but we still have to run these;
```
import intel_extension_for_pytorch as ipex
from infer.modules.ipex import ipex_init
ipex_init()
```
Same thing for the rest.

Reply:
> I can replace try with if has_xpu but we still have to run these;
> 
> ```
> import intel_extension_for_pytorch as ipex
> from infer.modules.ipex import ipex_init
> ipex_init()
> ```
> 
> Same thing for the rest.

Evev if a py file import the config.py and config.py had executed those?